ABOUT.md

```diff
@@ -1,45 +1,90 @@
+___
+MineColonies is a free and open source mod developed by Let's Dev Together (LDT), a non-profit community. The source code is available on [GitHub](https://github.com/ldtteam/minecolonies). Our developers are a hard-working, well-integrated coding team, continuously adding more content to make the MineColonies experience even greater. However, we are always looking for more people to contribute to the mod, whether as a coder, builder, artist, voice actor, wiki editor, tester, or simply supporting us on [Patreon](https://https://www.patreon.com/minecolonies)!
 
-### Have fun, Play with Friends
-Minecolonies has a very powerful multi-player system and an awesome tool for learning about Town planning, building and especially management. The permission system allows for collaborative team play in multi-player where you can assign different roles in a Town such as; Officer, Friend Neutral or even Hostile! Also have individual competing colonies at the same time, all with a strong protection system for server management and Anti-Griefing. Fun for all! 
+Found a bug? Report it as an [issue](https://https://github.com/ldtteam/minecolonies/issues/new/choose) to help us give you the best gaming experience. If you have any questions or just want to join the conversation, check us out on [Discord](https://discord.minecolonies.com)!
 ___
-##### Minecolonies is a free and open source mod developed by LDT (Let's Dev Together), a non-profit community. The source code is on [GitHub.](https://github.com/ldtteam/minecolonies) Our developers are a hardworking well integrated coding team, continuously working to add more content to our mod and make the Minecolonies experience even greater! Contribute to our Project. Found any bug? Report an [issue](https://github.com/ldtteam/minecolonies/issues/new) so as to help us ensure you have the best gaming experience. For more information and enquiries, contact us on [Discord.](https://discord.gg/YEas2Yv)
-___ 
 
 ## Our Team
 
 ### Project Leads
-* Raycoms – Programming and Management
-* Kostronor – Programming and Management
-* Asherslab – Programming and Management
-* OrionOnline - Programming and Management, Build System Engineer 
-* Pmardle – Project Management and Ideas (inactive)
+* Raycoms – Lead Lead Dev, Programming, and Active Maintainer
+* AshersLab – Programming and Active Maintainer
+* OrionOnline - Programming, Build System Engineer, and Active Maintainer
+* Someaddons - Programming, Pathfinding, Performance, and Active Maintainer
+* Pmardle – Founder, Project Management (inactive)
 * Flaminyogurt – Programming and Management (inactive)
+* Kostronor – Programming and Management (inactive)
 
-### Programming 
-* Xavier
+### Programming
```

Comment:
Might be a bit less specific on these, most of these guys do other stuff too lol. plus less specific means we don't need to update it as often =P

Reply:
```
### Programming
* Gattsuru - Research System Development and General Programming
* JiaYow - General Programming
* Mekle - Recipe System Development and General Programming
* Minecraftschurli - Worker Development and General Programming
* Nightenom - Structurize Development and General Programming
* PowerSchill - General Programming
* ShadowProtocol - UI/UX Development, Linguistics, and General Programming
* ToMe25 - General Programming
```
like so?

Reply:
yup looks better